src/pages/workspace/NewWorkspacePage.js

```diff
@@ -0,0 +1,144 @@
+                                                    {translate('workspace.new.editPhoto')}
+                                                </Text>
+                                            </View>
+                                        </View>
+                                    )}
+                                />
+                                <CreateMenu
+                                    isVisible={this.state.isEditPhotoMenuVisible}
+                                    onClose={() => this.setState({isEditPhotoMenuVisible: false})}
+                                    onItemSelected={() => this.setState({isEditPhotoMenuVisible: false})}
+                                    menuItems={this.createMenuItems(openPicker)}
+                                    anchorPosition={styles.createMenuPositionProfile}
+                                    animationIn="fadeInRight"
+                                    animationOut="fadeOutRight"
+                                />
+                            </>
+                        )}
+                    </AttachmentPicker>
+
+                    <View style={[styles.mt6, styles.w100, styles.flex1]}>
+                        <TextInputWithLabel
+                            label={translate('workspace.new.chooseAName')}
+                            value={this.state.name}
+                            onChangeText={name => this.setState({name})}
+                        />
+                        <Text style={[styles.mt6, styles.textP]}>{translate('workspace.new.helpText')}</Text>
+                        <View
+                            style={[
+                                styles.mt3,
+                                styles.flexRow,
+                                styles.mb6,
+                                styles.justifyContentBetween,
+                                styles.alignItemsCenter,
+                            ]}
+                        >
+                            <View style={styles.flex4}>
+                                <Text style={styles.textMicro}>
+                                    {translate('workspace.new.requestCall')}
+                                </Text>
+                            </View>
+                            <View style={[styles.flex1, styles.alignItemsEnd]}>
+                                <Switch
+                                    isOn={this.state.requestCall}
+                                    onToggle={requestCall => this.setState({requestCall})}
+                                />
+                            </View>
+                        </View>
+                    </View>
+
+                    <Button success style={[styles.w100]} text={this.props.translate('workspace.new.getStarted')} />
+                </View>
+            </ScreenWrapper>
+        );
+    }
+}
+
+NewWorkspacePage.propTypes = propTypes;
+NewWorkspacePage.displayName = 'NewWorkspacePage';
```

Comment:
NAB, class components do not require this `displayName`

Reply:
True!